lib/server/ServerRenderer.js

```diff
@@ -84,6 +86,10 @@ function insertBaseTag(html, appRoot, serverRequest) {
     return htmlWithBrisketBaseTag;
 }
 
+function stripIllegalCharacters(input) {
```

Comment:
When these characters are stringified on the page in bootstrap data, it breaks the `clientApp.start`
They are line terminators in javascript string, but not in JSON